styles/effects.css

```diff
@@ -10,29 +10,3 @@ h1.special {
     md:leading-[4rem]
     md:-tracking-[0.06rem];
 }
-
-div.glowingBackdrop {
-  @apply absolute
-    left-0
-    top-0
-    -z-10
-    size-full
-    bg-[url('/static/images/patterns/hexagon-grid.svg')]
```

Comment:
Is this image being used elsewhere, or can it be removed as part of this?

Reply:
Good point

Reply:
it looks to be used in `Preview`